### DISCUSSION TITLE
Questions about the Input_selects used in lovelace.yaml file

### DISCUSSION THREAD

**Discussion by abhayms-git** [re-1st/HA-3d-floorplan#2] [General]
Hello,

Thanks for sharing you code for the floorplan dashoard. I am just trying to understand and learn how you have developed the dashboard. Few questions:
1)in the lovelace_ui.yaml, there are lot of input_selcts that are used. For example: right at the start based on the weather condition you load a different floorplan image. 
Question 1): Where are these input_selects defined? Did you do that in the helper settings? Could you please load those settings to GIT as 
well?
Question 2): Based on weather change like rain or midday, how did you change that specific input_select?
2) How have you loaded the single file of lovelace_ui.yaml to show up all the floors? i see that single file has navigation path to different floors. But where in the UI do you click to go to different floors? i checked your video but cannot seem to figure out

**Comment by re-1st** [comment]
hey,
thanks for your message. I'm sorry for the confusion, The input selects are defined in node red, I should have described this in the readme (EDIT: i did now)
I left out the node red config because i was unsure about security issues caused by nodes containing passwords. If you use Node red, i could send you the flow which specificly controls the weather input select if you like?

2) i just click on the stairs and doors. if you change the opacity in the color of the button of the navigation, it will reveal itself. 

Like in this case: 

          - type: "custom:button-card"
            aspect_ratio: 2/1
            show_name: false

            #change this>>>>
            color: "rgba(255, 0 , 0 , 0)"
            # into this:>>>>
            color: "rgba(255, 0 , 0 , 1)"

            color_type: card
            tap_action:
              action: navigate
              navigation_path: /lovelace/2
              haptic: success
            style:
              left: 37%
              top: 52%
              width: 18%
